src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -411,6 +413,9 @@ public FormValidation doCheckServer(@QueryParameter String value, @QueryParamete
             if(!Jenkins.get().hasPermission(Jenkins.ADMINISTER))
                 return FormValidation.ok();
 
+            if(FIPS140.useCompliantAlgorithms() && managerPassword.length() < 14)
```

Comment:
Checking that we are running Jenkins in FIPS mode and that the password is FIPS compliant (see https://github.com/jenkinsci/jep/tree/master/jep/237 for Jenkins FIPS support)